recipes/clangdev/build.sh

```diff
@@ -0,0 +1,11 @@
+mkdir build
+cd build
+
+cmake \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DLLVM_ENABLE_RTTI=ON \
+  -DBUILD_SHARED_LIBS=ON \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  ..
+cmake --build . -- -j 8
```

Comment:
Done, although oversubscribing is not usually an issue in my experience--i.e. 8 is fine even for single-core, assuming there's sufficient RAM.

Made the same change in `llvmdev`, #1482.